test/e2e/api/matchMedia/map-bounding-box.html

```diff
@@ -0,0 +1,101 @@
+                const query = `(map-projection: OSMTILE) and (map-zoom < 14) and (map-top-left-easting < ${bottomRightEasting}) and (map-bottom-right-easting > ${topLeftEasting}) and (map-bottom-right-northing < ${topLeftNorthing}) and (map-top-left-northing > ${bottomRightNorthing})`;
+
+                let matcher = map.matchMedia(query);
+                const logResults = (e) => {
+                    if (e.target.matches) {
+                        layer.checked = true;
+                        layer.removeAttribute('hidden');
+                    } else {
+                        layer.checked = false;
+                        layer.hidden = true;
+                    }
+                };
+                matcher.addEventListener('change', logResults);
+
+                // create a layer to visually represent the query as the map moves
+                let f = `<map-layer checked label="test media query"><map-meta name="projection" content="OSMTILE"></map-meta>
+                    <map-meta name="cs" content="pcrs"></map-meta><map-feature><map-properties>${query}</map-properties>
+                    <map-geometry><map-polygon><map-coordinates>${topLeftEasting} ${topLeftNorthing}
+                    ${bottomRightEasting} ${topLeftNorthing} ${bottomRightEasting} ${bottomRightNorthing} ${topLeftEasting} ${bottomRightNorthing}
+                    ${topLeftEasting} ${topLeftNorthing}</map-coordinates</map-polygon></map-geometry></map-feature></map-layer>`;
+
+                const parser = new DOMParser();
+                const layer = parser
+                    .parseFromString(f, 'text/html')
+                    .querySelector('map-layer');
+                map.appendChild(layer);
+                return { matcher, logResults };
+            }
+            document.querySelector('.getBoundingBox').addEventListener('click', getBoundingBox);
+        });
+    </script>
+
+    <mapml-viewer projection="OSMTILE" zoom="14" lat="45.406314" lon="-75.6883335" controls=""
+        controlslist="geolocation">
+        <map-layer data-testid="osm-layer" label="OpenStreetMap" checked="">
+            <map-link rel="license" title="Â© OpenStreetMap contributors CC BY-SA"
+                href="https://www.openstreetmap.org/copyright"></map-link>
+            <map-extent units="OSMTILE" checked="checked">
+                <map-input name="z" type="zoom" value="18" min="0" max="18"></map-input>
+                <map-input name="x" type="location" units="tilematrix" axis="column" min="0" max="262144"></map-input>
+                <map-input name="y" type="location" units="tilematrix" axis="row" min="0" max="262144"></map-input>
+                <map-link rel="tile" tref="https://tile.openstreetmap.org/{z}/{x}/{y}.png"></map-link>
+            </map-extent>
+        </map-layer>
+    </mapml-viewer>
+
+    <div class="description">
+        <button class="getBoundingBox">Get Bounding Box</button>
```

Comment:
I would say this is "the hard way" because it makes you have to click the button in the test.  The DOMContentLoaded event handler could do it (although the viewer does take some time to get ready, so maybe the button clicking just works around that?).